model-optimizer/extensions/middle/FakeSplitOutputs.py

```diff
@@ -31,13 +31,20 @@ def pattern():
     @staticmethod
     def replace_pattern(graph: Graph, match: dict):
         node = match['op']
+        AddFakeOutputsToSplit.split_normalize_outputs(node)
 
+    '''
+    For IR Reader we need to call adding fake outputs without pattern matcher.
+    '''
```

Comment:
We do not leave comments in such a format. Please, add a comment under the `def split_normalize_outputs(node: Node):`. Also, the comment should be left in the MO IR Reader code instead of this place because it is applicable for MO IR Reader.

Also describe in the comment why you need to call this method in the MO IR Reader.